sdk/authorization/Azure.ResourceManager.Authorization/tests/Scenario/DenyAssignmentCollectionTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Resources;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Authorization.Tests.Scenario
+{
+    public class DenyAssignmentCollectionTests : AuthorizationManagementTestBase
+    {
+        public DenyAssignmentCollectionTests(bool isAsync)
+                    : base(isAsync)//, RecordedTestMode.Record)
+        {
+        }
+
+        public ResourceGroupResource ResourceGroup { get; set; }
+
+        [Test]
+        public async Task Get()
+        {
+            var randomGuid = Recording.Random.NewGuid().ToString();
+            var scope = DefaultSubscription.Id;
+            var denyAssignmentCollection = Client.GetDenyAssignments(scope);
+            try
+            {
+                var denyAssignment = (await denyAssignmentCollection.GetAsync(randomGuid)).Value;
+                Assert.AreEqual(1, 0);
+            }
+            catch (RequestFailedException ex) when (ex.Status == 404)
+            {
+                Assert.AreEqual(1, 1);
+            }
+        }
+
+        [Test]
+        public async Task GetAllBySubscription()
+        {
+            var scope = DefaultSubscription.Id;
+            var denyAssignmentCollection = Client.GetDenyAssignments(scope);
+            var denyAssignments = await denyAssignmentCollection.GetAllAsync().ToEnumerableAsync();
+            Assert.AreEqual(0, denyAssignments.Count);
```

Comment:
use `Assert.IsEmpty(denyAssignments)` instead, same for the other tests

Reply:
Resolved